README.md

```diff
@@ -1,3 +1,108 @@
-# oscker
+
+# Oscker
 
 Oscker aim to make a generic descriptor to make images for different CSPs.
+
+## Run
+
+Specify the cloud vendor and basic image by calling the getHclScript method in Main to generate the hcl script executed by packer
+
+### input param
+
+```json
+{
+    "cloudType": "huaweicloud",
```

Comment:
Should we make the "cloudType" a parameter in the command line, but not a configuration?